src/ngraph/runtime/gpu/cublas_emitter.hpp

```diff
@@ -0,0 +1,65 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <functional>
```

Comment:
Way too many includes here. I think you need perhaps 3 to satisfy the header's requirements. Don't include things not directly used by the header.
All <> headers in one block and "" headers in a separate block. Don't use newlines inside a block

Reply:
@rkimballn1 BTW, will enable `IncludeBlocks` option in clang-format help?

Reply:
Unfortunately `IncludeBlocks` is not available in clang-format-3.9 :(